src/DGtal/helpers/Shortcuts.h

```diff
@@ -1896,12 +1896,14 @@ namespace DGtal
 	      }
 	    }
 	  // Simplify materials (very useful for blender).
-          Idx j = 0;
           std::map<Color,Idx> map_colors;
-          for ( auto && c : diffuse_colors )
-            if ( ! map_colors.count( c ) )
-              map_colors[ c ] = j++;
-
+	  {
+            Idx j = 0;
+            for ( auto && c : diffuse_colors )
```

Comment:
@JacquesOlivierLachaud is not that in DGtal the convention is to name instances and variables in camel case? Anyways, this is a general question for these files and not for the PR itself.

Reply:
Camel case is for classes and methods. I don't remember that we had camel case conventions for local variables and I would not be in favour of it.